src/connectors/index.ts

```diff
@@ -10,6 +10,7 @@ import { ChainId } from '../constants'
 
 const RPC = {
     [ChainId.MAINNET]: 'https://rpcapi.fantom.network',
+    [ChainId.TESTNET]: 'https://rpc.testnet.fantom.network/',
```

Comment:
I don't see the point of having testnet inside the codebase, maybe people can get confused